codalab/server/worker_info_accessor.py

```diff
@@ -33,6 +33,8 @@ def _fetch_workers(self):
             for uuid in worker['run_uuids']:
                 self._uuid_to_worker[uuid] = worker
             self._user_id_to_workers[worker['user_id']].append(worker)
+            # Add an additional 'has_gpus' flag to indicate if the current worker has GPUs or not
+            worker['has_gpus'] = True if worker['gpus'] > 0 else False
```

Comment:
Add a comment that `gpus` is the number of free GPUs in general, so this isn't just trivial.